src/frontends/tensorflow/src/frontend.cpp

```diff
@@ -178,6 +182,8 @@ bool FrontEnd::supported_impl(const std::vector<ov::Any>& variants) const {
         } else if (GraphIteratorProtoTxt::is_supported(model_path)) {
             // handle text protobuf format
             return true;
+        } else {
+            throw_path_error(ov::util::wstring_to_string(model_path));
```

Comment:
What about overriding `throw_path_error` to accept also `wstring`?

Reply:
Added